functions/Run_TA.m

```diff
@@ -163,7 +163,12 @@
             % Motion Analysis
             %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
             if param.doScrubbing
-                param.TemporalMask = AssessMotion(SubjPath_TA,i_TA,param,fid);
+                if isempty(param.TemporalMask_user) 
+                    param.TemporalMask = AssessMotion(SubjPath_TA,i_TA,param,fid);
+                else
+                    param.TemporalMask = param.TemporalMask_user;
+                     WriteInformation(fid,'User temporal mask used');
+                end
```

Comment:
This was in previous versions of iCAPs I use it and they should let people put their own temporal mask

Reply:
@eurunuela should I delete this too?